lib/quantcast/index.js

```diff
@@ -177,17 +178,25 @@ Quantcast.prototype.completedOrder = function(track){
 Quantcast.prototype._labels = function(type){
   var args = Array.prototype.slice.call(arguments, 1);
   var advertise = this.options.advertise;
+  var separator = advertise ? ' ' : '.';
+  var customVars;
 
   if (advertise && 'page' == type) type = 'event';
   if (advertise) type = '_fp.' + type;
 
-  var separator = advertise ? ' ' : '.';
   var ret = reduce(args, function(ret, arg){
-    if (arg != null) {
+    if (is.object(arg) && arg.label) {
+      customVars = arg.label;
+    }
```

Comment:
I don't really like how I'm doing this, but after trying about four other ways, this is what I settled on.